internal/controller/stas/scan_job_controller.go

```diff
@@ -201,7 +201,7 @@ func (r *ScanJobReconciler) reconcileJob(ctx context.Context, job *batchv1.Job)
 	if err != nil {
 		switch {
 		case staserrors.IsJobPodNotFound(err), staserrors.IsScanJobContainerWaiting(err):
-			logf.FromContext(ctx).V(3).Info("Failed to get job %q logs %s", job.Name, err)
+			logf.FromContext(ctx).V(-3).Info("Failed to get job logs.", "error", err)
```

Comment:
```suggestion
			logf.FromContext(ctx).V(-3).Error(err, "While getting logs")
```

Reply:
i think it should be ok

Reply:
Error dumps stacktrace which is probably not required

```
  2023-01-26T11:16:01+01:00     ERROR   While getting logs      {"controller": "job", "controllerGroup": "batch", "controllerKind": "Job", "Job": {"name":"application-example-app-a309a-ab8ae","namespace":"image-scanner-jobs"}, "namespace": "image-scanner-jobs", "name": "application-example-app-a309a-ab8ae", "reconcileID": "9afb873e-467d-420d-b865-
8cf6d0e41539", "error": "test error"}
  github.com/statnett/image-scanner-operator/internal/controller/stas.(*ScanJobReconciler).reconcileJob
        /home/sunils/sandbox/padlar-iso/internal/controller/stas/scan_job_controller.go:220
  github.com/statnett/image-scanner-operator/internal/controller/stas.(*ScanJobReconciler).reconcile.func1.1
        /home/sunils/sandbox/padlar-iso/internal/controller/stas/scan_job_controller.go:164
  github.com/statnett/image-scanner-operator/internal/controller.Reconcile
        /home/sunils/sandbox/padlar-iso/internal/controller/reconcile.go:16
  github.com/statnett/image-scanner-operator/internal/controller/stas.(*ScanJobReconciler).reconcile.func1
        /home/sunils/sandbox/padlar-iso/internal/controller/stas/scan_job_controller.go:167
  sigs.k8s.io/controller-runtime/pkg/reconcile.Func.Reconcile
        /home/sunils/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.1/pkg/reconcile/reconcile.go:102
  sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Reconcile
        /home/sunils/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.1/pkg/internal/controller/controller.go:122
  sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler
        /home/sunils/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.1/pkg/internal/controller/controller.go:323
  sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
        /home/sunils/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.1/pkg/internal/controller/controller.go:274
  sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
        /home/sunils/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.1/pkg/internal/controller/controller.go:235
```

will revert this back to info